src/one_dragon/base/controller/pc_controller_base.py

```
@@ -185,11 +411,75 @@ def drag_to(self, end: Point, start: Point = None, duration: float = 0.5):
 
-    def close_game(self):
-        """
-        关闭游戏
-        :return:
+    def background_drag(self, start: Point | None, end: Point, duration: float = 0.5) -> bool:
+        """后台拖拽：SetCursorPos 移动鼠标 + PostMessage 按住拖动。
+
+        使用 SetCursorPos 物理移动鼠标，分步插值模拟拖拽轨迹。
+
+        Args:
+            start: 拖拽起点（游戏坐标），None 时使用当前鼠标位置
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)